session.go

```diff
@@ -1051,25 +1058,7 @@ func (q *Query) Release() {
 
 // reset zeroes out all fields of a query so that it can be safely pooled.
 func (q *Query) reset() {
-	q.stmt = ""
-	q.values = nil
-	q.cons = 0
-	q.pageSize = 0
-	q.routingKey = nil
-	q.routingKeyBuffer = nil
```

Comment:
looking at this, we could probably keep these buffers between pooling but would need to verify that they are not kept around anywhere after being used.